index.html

```diff
@@ -994,6 +994,65 @@
 Issue(16): This description of what makes information sensitive still needs to
 be refined.
 
+## Unwanted profiling {#hl-unwanted-profiling}
+
+Contributes to [=surveillance=], [=correlation=], [=identification=], and
+singling-out / discrimination.
+
+Unwanted profiling occurs when a site is able to learn attributes or
+characteristics about a person, that a) the site visitor did not intend the site
+to learn, and b) the site visitor reasonably could not anticipate a
+site would be able to learn.
+
+Profiling contributes to, but is distinct from, other privacy risks discussed
+in this document.
+For example, unwanted profiling may contribute to
+[[[#hl-recognition-same-site]]], by adding stable and semi-identifying
+information that can contribute to <a
+data-cite="fingerprinting-guidance#dfn-browser-fingerprinting">browser
+fingerprinting</a>. Unwanted profiling is distinct from unwanted
+recognition though, in that a person may wish to not share some kinds
+of information about themselves <em>even in the presence of guarantees
+that such information will not lead to them being re-identified</em>.
+
+Similarly, unwanted profiling is related to [[[#hl-sensitive-information]]],
```

Comment:
I wonder if this sentence/paragraph can be simplified a little. I don't have concrete examples right now though.